tests/main/interfaces-network-control/task.yaml

```diff
@@ -20,12 +20,8 @@ environment:
     ARP_ENTRY_ADDR: "30.30.30.30"
 
 prepare: |
-    #shellcheck source=tests/lib/snaps.sh
-    . "$TESTSLIB/snaps.sh"
-    #shellcheck source=tests/lib/systemd.sh
-    . "$TESTSLIB/systemd.sh"
```

Comment:
systemd was unused here?

Reply:
It seems so, but let's have it confirmed by Sergio.

Reply:
Yes, it is unused, you can expand this section just a little below to see.